server/config.ts

```diff
@@ -177,8 +177,19 @@ export default {
   ),
   featureToggles: {
     // someToggleEnabled: toBoolean(get('SOME_TOGGLE_ENABLED', false)),
-    reviewsPrisonsEnabled: get('REVIEWS_PRISONS_ENABLED', ''),
     completedGoalsEnabled: toBoolean(get('COMPLETED_GOALS_ENABLED', false)),
     archiveGoalNotesEnabled: toBoolean(get('ARCHIVE_GOAL_NOTES_ENABLED', false)),
+    reviewJourneyEnabledForPrison: (prisonId: string): boolean => {
```

Comment:
`reviewJourneyEnabledForPrison` is a refactor of the previous property `reviewsPrisonsEnabled`, and it means that the client code can simply call the function, rather than having to worry about do the splitting etc itself (eg. see the change to `server/routes/reviewPlan/index.ts` below 👇 )